releasenotes/notes/48603.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+releaseNotes:
+- |
+  **Added** completion for Kubernetes pods for some commands that can select pods, such as `istioctl proxy-config <pod>`.`
```

Comment:
typo?  I don't see the `completion` library changes in `istioctl/pkg/proxyconfig/proxyconfig.go`, did you mean `istioctl proxy-status`?

Reply:
Yes this is a typo, thanks!